File: WebApp/Models/BC/BC_MANAGER.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class BC_MANAGER : JCLib.Mvc.BaseModel
    {
        public string WH_NO { get; set; }
        public string MANAGERID { get; set; }
        public string MANAGERNM { get; set; }
        public string USERID { get; set; }
        public string CREATE_DATE { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_DATE { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
        public string MNAME { get; set; }
        public string CNT { get; set; }
        public string DISPLAY_WHNO { get; set; }//白底顯示
        public string TEXT_WHNO { get; set; }//紅底顯示
        public string DISPLAY_MANAGERID { get; set; }//白底顯示
        public string TEXT_MANAGERID { get; set; }//紅底顯示
        public string TEXT_MANAGERNM { get; set; } //紅底顯示
    }
}

File: WebApp/Scripts/app/model/BcManager.js
Ext.define('WEBAPP.model.BcManager', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'WH_NO', type: 'string' },
        { name: 'MANAGERID', type: 'string' },
        { name: 'MANAGERNM', type: 'string' },
        { name: 'USERID', type: 'string' },
        { name: 'CREATE_DATE', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_DATE', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },
        { name: 'MNAME', type: 'string' },
        { name: 'CNT', type: 'string' },
        { name: 'DISPLAY_WHNO', type: 'string' },
        { name: 'TEXT_WHNO', type: 'string' },
        { name: 'DISPLAY_MANAGERID', type: 'string' },
        { name: 'TEXT_MANAGERID', type: 'string' },
        { name: 'TEXT_MANAGERNM', type: 'string' }
    ]
});